force-app/main/default/classes/OpportunityTriggerHandler.cls

```diff
@@ -0,0 +1,171 @@
+public with sharing class OpportunityTriggerHandler extends TriggerHandler{
```

Comment:
This is a great TriggerHandler and can be used to model your AccountTriggerHandler after